src/modules/extractors/recipes.extractor.ts

```diff
@@ -211,7 +211,13 @@ export class RecipesExtractor {
     const ticks = isNaN(Number(recipeProperties.ticks))
       ? null
       : Number(recipeProperties.ticks);
-
+    let toolIds: number[] = [];
```

Comment:
can use const

Reply:
We assign to it in the if statement, so it needs to be mutable.

Reply:
Ah true, nevermind 🤦‍♂️